<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPassword</class>
 <widget class="QWidget" name="ResetPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: black;
background-color: #283035;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="reset_password">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="username_page">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>70</number>
         </property>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_next">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>285</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>285</width>
             <height>37</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: rgba(255, 98, 71, 0.6);
color: white; 
font:bold;
padding: 8px 12px 8px 12px;
border: 0px;
}

QPushButton:hover {
	background-color: rgba(255, 98, 71, 1);
}</string>
           </property>
           <property name="text">
            <string>Recover Username</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="lineEdit_username">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>285</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>285</width>
             <height>37</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #545454;
color:white;
border-style:none;</string>
           </property>
           <property name="placeholderText">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_username_invalid">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>15</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="security_question_page">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="bottomMargin">
          <number>70</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="random_question_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>285</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>285</width>
             <height>37</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #545454;
color:white;
border-style:none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_question_invalid">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="lineEdit_answer">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>285</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>285</width>
             <height>37</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #545454;
color:white;
border-style:none;</string>
           </property>
           <property name="placeholderText">
            <string>Answer</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>15</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pushButton_submit_answer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>285</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>285</width>
             <height>37</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: rgba(255, 98, 71, 0.6);
color: white; 
font:bold;
padding: 8px 12px 8px 12px;
border: 0px;
}

QPushButton:hover {
	background-color: rgba(255, 98, 71, 1);
}</string>
           </property>
           <property name="text">
            <string>Submit Answer</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>15</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="code_page">
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>45</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>90</number>
       </property>
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit_email_code">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>285</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>285</width>
           <height>37</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #545454;
color:white;
border-style:none;</string>
         </property>
         <property name="placeholderText">
          <string>Email Code</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_verify_code">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>285</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>285</width>
           <height>37</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: rgba(255, 98, 71, 0.6);
color: white; 
font:bold;
padding: 8px 12px 8px 12px;
border: 0px;
}

QPushButton:hover {
	background-color: rgba(255, 98, 71, 1);
}</string>
         </property>
         <property name="text">
          <string>Verify</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="reset_password_page">
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>40</number>
       </property>
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit_new_pass">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>285</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>285</width>
           <height>37</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #545454;
color:white;
border-style:none;</string>
         </property>
         <property name="placeholderText">
          <string>New Password</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_confirm_pass">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>285</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>285</width>
           <height>37</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #545454;
color:white;
border-style:none;</string>
         </property>
         <property name="placeholderText">
          <string>Confirm New Password</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_reset_password">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>285</width>
           <height>37</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: rgba(255, 98, 71, 0.6);
color: white; 
font:bold;
padding: 8px 12px 8px 12px;
border: 0px;
}

QPushButton:hover {
	background-color: rgba(255, 98, 71, 1);
}</string>
         </property>
         <property name="text">
          <string>Reset Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_username</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_next</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_answer</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_submit_answer</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_email_code</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_verify_code</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_confirm_pass</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_reset_password</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
